dev-notes/JavaSDK_With_CanarySubscriptions.md

```diff
@@ -0,0 +1,50 @@
+
+### Using Java SDK with Canary enabled subscriptions
+
+
+There are two canary regions:
+
+```
+Central US EUAP
+East US 2 EUAP
+```
+
+If you have an Azure subscription whitelisted for Canary then you can use it as usual with canary regions.
```

Comment:
Suggestion :
It would be nice to make it clear that they need a whitelisted subscription and help the customer with a link to instructions on how to get whitelisted.I would reword this sentence something like :
```suggestion
You need to have a subscription whitelisted to be able to work with Canary regions. Go [here](<link to the site here>) for instructions on how to get a subscription whitelisted.
```

Reply:
thanks for rewording, I incorporated it but as discussed offline link is not added.